src/plugins/index_pattern_management/public/components/create_button/create_button.tsx

```diff
@@ -48,26 +46,21 @@ interface Props {
     isBeta?: boolean;
     onClick: () => void;
   }>;
-  uiCapabilities: UICapabilities;
 }
 
-class CreateButtonComponent extends Component<Props, State> {
+export class CreateButton extends Component<Props, State> {
   public state = {
     isPopoverOpen: false,
   };
 
   public render() {
-    const { options, children, uiCapabilities } = this.props;
+    const { options, children } = this.props;
     const { isPopoverOpen } = this.state;
 
     if (!options || !options.length) {
       return null;
     }
 
-    if (!uiCapabilities.indexPatterns.save) {
-      return null;
-    }
-
```

Comment:
Why did this get removed?

Reply:
Instead of having the element determine if it renders, the parent element decides.